src/theme/util.ts

```diff
@@ -0,0 +1,36 @@
+import { bp } from './breakpoints';
+
+const relativeBP = {
+  xs: { max: (bp.sm - 1) / 16, min: -Infinity },
+  sm: { max: (bp.md - 1) / 16, min: bp.sm / 16 },
+  md: { max: (bp.lg - 1) / 16, min: bp.md / 16 },
+  lg: { max: (bp.xl - 1) / 16, min: bp.lg / 16 },
+  xl: { max: (bp.xxl - 1) / 16, min: bp.xl / 16 },
+  xxl: { max: (bp.xxl2 - 1) / 16, min: bp.xxl / 16 },
+  xxl2: { max: Infinity, min: bp.xxl2 / 16 },
+};
```

Comment:
Probably worth creating a variable for self-documentation and in case for some reason we were to change the base size.

```suggestion
const BASE_FONT = 16;

const relativeBP = {
  xs: { max: (bp.sm - 1) / BASE_FONT, min: -Infinity },
  sm: { max: (bp.md - 1) / BASE_FONT, min: bp.sm / BASE_FONT },
  md: { max: (bp.lg - 1) / BASE_FONT, min: bp.md / BASE_FONT },
  lg: { max: (bp.xl - 1) / BASE_FONT, min: bp.lg / BASE_FONT },
  xl: { max: (bp.xxl - 1) / BASE_FONT, min: bp.xl / BASE_FONT },
  xxl: { max: (bp.xxl2 - 1) / BASE_FONT, min: bp.xxl / BASE_FONT },
  xxl2: { max: Infinity, min: bp.xxl2 / BASE_FONT },
};
```